app/src/androidTest/java/com/eis/audioUtility/AudioUtilityManagerTest.java

```diff
@@ -0,0 +1,202 @@
+package com.eis.audioUtility;
+
+/**
+ * @author Francesco Bau', Alberto Ursino
+ */
+
+import android.content.Context;
+
+import androidx.test.platform.app.InstrumentationRegistry;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class AudioUtilityManagerTest {
+
+    Context context;
+
+    @Before
+    public void init() {
+        context = InstrumentationRegistry.getInstrumentation().getTargetContext();
+    }
+
+    @Test
+    public void setAlarmVolume_percentage_isTooLow() {
+        try {
+            AudioUtilityManager.setVolume(context, AudioUtilityManager.AUMStream.ALARM, -1);
+            Assert.fail("Should have thrown IllegalArgumentException.");
+        } catch (IllegalArgumentException e) {
+            //Success
+        } catch (Exception x) {
+            Assert.fail(x.getStackTrace() + "\nShould have thrown IllegalArgumentException");
+        }
+    }
+
+    @Test
+    public void setAlarmVolume_percentage_isTooHigh() {
+        try {
+            AudioUtilityManager.setVolume(context, AudioUtilityManager.AUMStream.ALARM, 101);
+        } catch (IllegalArgumentException e) {
+            // Success
+        } catch (Exception x) {
+            Assert.fail(x.getStackTrace() + "\nShould have thrown IllegalArgumentException");
+        }
+    }
+
+    @Test
+    public void setAlarmVolumeToMax_maxVolume_isOK() {
+        try {
+            AudioUtilityManager.setVolumeToMax(context, AudioUtilityManager.AUMStream.ALARM);
+            //Success
+        } catch (IllegalArgumentException e) {
+            Assert.fail(e.getStackTrace() + "\nMAX_PERCENTAGE is NOT OK. Please check this value.");
+        } catch (Exception x) {
+            Assert.fail(x.getStackTrace() + "\nIt's NOT supposed to fail like this!");
+        }
+    }
+
+    @Test
+    public void setAlarmVolumeToMin_minVolume_isOK() {
+        try {
+            AudioUtilityManager.setVolumeToMin(context, AudioUtilityManager.AUMStream.ALARM);
+            //Success
+        } catch (IllegalArgumentException e) {
+            Assert.fail(e.getStackTrace() + "\nMIN_PERCENTAGE is NOT OK. Please check this value.");
+        } catch (Exception x) {
+            Assert.fail(x.getStackTrace() + "\nIt's NOT supposed to fail like this!");
+        }
+    }
+
+    @Test
+    public void getCurrentAlarmVolume_method_isOK() {
+        try {
+            AudioUtilityManager.getVolume(context, AudioUtilityManager.AUMStream.ALARM);
+            //Success
+        } catch (Exception e) {
```

Comment:
Test fails automatically if an exception is thrown

Reply:
ok.. So what am I supposed to do? Should I delete this test?

Reply:
Yes, you can delete the try-catch and just leave the AudioUtility line

Reply:
Should I do the same to all these methods with catch(Exception e) declaration?